src/modules/ekf2/EKF/python/ekf_derivation/derivation.py

```diff
@@ -390,6 +390,24 @@ def compute_mag_z_innov_var_and_h(
 
     return (innov_var, H.T)
 
+def compute_yaw_innov_var_and_h(
```

Comment:
Would it overall be simpler if we just computed the innovation in here as well?

Reply:
I don't think there will be much computational benefit in doing so; it's just a couple of multiplications and additions to get the yaw innovation from 2 quaternions